peppy/project.py

```diff
@@ -1,10 +1,12 @@
 """
 Build a Project object.
 """
+from __future__ import annotations
+
```

Comment:
This will not work with Python versions below 3.10. I'm wondering is there a chance that something gets broken?

Reply:
I think we should delete it than!

Reply:
yes that would be a problem... this will show up in the unit tests, right?

Reply:
I'm unfamiliar with this. I'm able to import it in python 3.9 on my system:
```console
Python 3.9.13 (main, May 19 2022, 13:48:47) 
[Clang 13.1.6 (clang-1316.0.21.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from __future__ import annotations
>>> annotations
_Feature((3, 7, 0, 'beta', 1), (3, 10, 0, 'alpha', 0), 16777216)
```

Reply:
There will be problems in python3.10. I have fixed it.